src/main/java/appeng/core/features/DamagedItemDefinition.java

```diff
@@ -32,7 +34,7 @@
 {
 	private final IStackSrc source;
 
-	public DamagedItemDefinition( IStackSrc source )
+	public DamagedItemDefinition( @Nonnull IStackSrc source )
```

Comment:
You generally define `internal` as `private` so any private method can use them, ctor and public methods are public thus callable from outside of the context. 

http://www.javapractices.com/topic/TopicAction.do?Id=102